imports/ui/components/home/landing.js

```diff
@@ -24,6 +24,19 @@ Template.landing.onCreated(function() {
   });
 });
 
+var currentPlaylist = function() {
+   return Playlists.findOne({
+    $where: function() {
+      return this.showDate.getYear() === new Date().getYear() &&
+             this.showDate.getMonth() === new Date().getMonth() &&
+             this.showDate.getDate() === new Date().getDate() &&
+             parseInt(this.startTime.split(":")[0]) <= new Date().getHours() &&
+             (parseInt(this.endTime.split(":")[0]) > new Date().getHours() ||
+             this.endTime === "00:00:00");
+    }
+  }
+)};
+
```

Comment:
This appears in another file, correct? My intention was to see this factored out into a utility/helper file, since it appeared in separate files

Reply:
It's been factored out into `imports/startup/lib/helpers.js`.